spec/integration/categories_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
```

Comment:
- [ ] Please add some more integration tests fo the other controllers. here I only see that of the categories controller.